Updates/Instances/544_magtheridon.sql

```diff
@@ -223,12 +111,67 @@ INSERT INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `position_x`, `posit
--- INSERT INTO `waypoint_path_name` (`PathId`, `Name`) VALUES
--- INSERT INTO `waypoint_path` (`PathId`, `Point`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `WaitTime`, `ScriptId`, `Comment`) VALUES
+
+INSERT INTO `spawn_group_spawn` (`Id`, `Guid`, `SlotId`) VALUES
+(@SGGUID+1, @CGUID+32, 0), -- Hellfire Warder
+(@SGGUID+1, @CGUID+31, 1), -- Hellfire Warder
+(@SGGUID+1, @CGUID+33, 2), -- Hellfire Warder
+(@SGGUID+2, @CGUID+37, 0), -- Hellfire Warder
+(@SGGUID+2, @CGUID+38, 1), -- Hellfire Warder
+(@SGGUID+2, @CGUID+39, 2), -- Hellfire Warder
+(@SGGUID+3, @CGUID+28, 0), -- Hellfire Warder
+(@SGGUID+3, @CGUID+29, 1), -- Hellfire Warder
+(@SGGUID+3, @CGUID+30, 2), -- Hellfire Warder
+(@SGGUID+4, @CGUID+34, 0), -- Hellfire Warder
+(@SGGUID+4, @CGUID+35, 1), -- Hellfire Warder
+(@SGGUID+4, @CGUID+36, 2); -- Hellfire Warder
+
+INSERT INTO `spawn_group_formation` (`Id`, `FormationType`, `FormationSpread`, `FormationOptions`, `PathId`, `MovementType`, `Comment`) VALUES
+(@SGGUID+4, 2, 3, 0, @SGGUID+4, 2, 'Magtheridon - Hellfire Warder (3) | Patrol 1');
+
+INSERT INTO `waypoint_path_name` (`PathId`, `Name`) VALUES
+(@SGGUID+4,'Magtheridon - Hellfire Warder (3) | Patrol 1');
+
+INSERT INTO `waypoint_path` (`PathId`, `Point`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `WaitTime`, `ScriptId`) VALUES
+(@SGGUID+4,1,-73.72693,47.355293,-0.38537598,100,0,0),
+(@SGGUID+4,2,-53.951275,58.999866,3.1846943,100,0,0),
+(@SGGUID+4,3,-28.083755,65.604164,15.94728,100,0,0),
+(@SGGUID+4,4,-3.498642,66.86735,22.654028,100,0,0),
+(@SGGUID+4,5,14.21773,57.512276,22.968313,100,0,0),
+(@SGGUID+4,6,29.301313,40.577045,33.289814,100,0,0),
+(@SGGUID+4,7,44.670013,16.09524,45.009865,100,0,0),
+(@SGGUID+4,8,44.010056,-11.347733,45.010532,100,0,0),
+(@SGGUID+4,9,33.572083,-28.368456,37.985077,100,0,0),
+(@SGGUID+4,10,14.805847,-53.501915,22.991356,100,0,0),
+(@SGGUID+4,11,-11.261587,-62.130146,22.562366,100,0,0),
+(@SGGUID+4,12,-37.538406,-59.60784,11.227191,100,0,0),
+(@SGGUID+4,13,-70.55042,-44.39177,-0.3772414,100,0,0),
+(@SGGUID+4,14,-37.538406,-59.60784,11.227191,100,0,0),
+(@SGGUID+4,15,-11.261587,-62.130146,22.562366,100,0,0),
+(@SGGUID+4,16,14.805847,-53.501915,22.991356,100,0,0),
+(@SGGUID+4,17,33.572083,-28.368456,37.985077,100,0,0),
+(@SGGUID+4,18,44.010056,-11.347733,45.010532,100,0,0),
+(@SGGUID+4,19,44.670013,16.09524,45.009865,100,0,0),
+(@SGGUID+4,20,29.301313,40.577045,33.289814,100,0,0),
+(@SGGUID+4,21,14.21773,57.512276,22.968313,100,0,0),
+(@SGGUID+4,22,-3.498642,66.86735,22.654028,100,0,0),
+(@SGGUID+4,23,-28.083755,65.604164,15.94728,100,0,0),
+(@SGGUID+4,24,-53.951275,58.999866,3.1846943,100,0,0);
+
+DELETE FROM worldstate_name WHERE Id = 4805;
+INSERT INTO `worldstate_name` (`Id`, `Name`) VALUES 
+(4805, 'Magtheridon - Trash Respawn');
+
+DELETE FROM conditions WHERE condition_entry BETWEEN @CONDITIONID+1 AND @CONDITIONID+9;
```

Comment:
typo, there is no CONDITIONID defined (not needed cause its only 1 condition for this instance